src/jsxer/jsxer.cpp

```diff
@@ -27,17 +26,18 @@ void prepend_header(string& code, JsxbinVersion jsxbin_version, bool unblind) {
                     "\n"
                     "* JSXBIN " + version + "\n";
 
-    if (unblind) {
-        header += "* Jsxblind Deobfuscation Enabled (EXPERIMENTAL)\n";
+    header += "* Options:\n";
+    if (options & kDO_Unblind) {
+        header += "* \tJsxblind Deobfuscation (EXPERIMENTAL)\n";
```

Comment:
```suggestion
        header += "* Jsxblind Deobfuscation (EXPERIMENTAL)\n";
```
Looks kind of strange with a tab there

Reply:
it's there to show a list of options enabled which reflects changes in the decompilation.

Reply:
ohhh I didn't see the `Options:` before it! my bad.